java/src/main/java/org/rocksdb/RawSstFileReaderIterator.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * <p>An iterator that yields a sequence of internal key/value pairs from a source.
+ * Multiple implementations are provided by this library.
+ * In particular, iterators are provided
+ * to access the contents of a Table or a DB.</p>
+ *
+ * <p>Multiple threads can invoke const methods on an RocksIterator without
+ * external synchronization, but if any of the threads may call a
+ * non-const method, all threads accessing the same RocksIterator must use
+ * external synchronization.</p>
+ *
+ * @see RocksObject
+ */
+ class RawSstFileReaderIterator extends SstFileReaderIterator {
```

Comment:
Did you intend for this class to be package-private scope? i.e. not usable for anyone outside `org.rocksdb`?

Reply:
Yeah I missed it, I fixed it.